tac/project-updates/2024/2024-Q4-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Iroha
+parent: 2024
+grand_parent: Project Updates
+---
+
+| Iroha 2: [website](https://iroha.tech/) | Iroha 2: [documentation](https://docs.iroha.tech/)  |
+| :---- | :---- |
+
+# Project Health
+
+In the past quarter, Iroha 2 has made solid progress. The Iroha 2  team has released a pre-MVP version which is [RC 22.2](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.2), and have been focusing on improving performance, refining the API, and updating SDKs and documentation. We're gearing up to release the MVP in the coming weeks, which will feature SDKs for [Java](https://github.com/hyperledger-iroha/iroha-java) and [Python](https://github.com/hyperledger-iroha/iroha-python); a [JavaScript/TS](https://github.com/hyperledger-iroha/iroha-javascript) SDK will follow in a later release.
+
+Additionally, the Iroha 2 team has finished developing the Iroha 2 [Blockchain Explorer](https://github.com/soramitsu/iroha2-block-explorer-web). The MVP is now in great shape and ready for use.
+
+We also successfully moved Iroha 2’s codebase to its [own repository](https://github.com/hyperledger-iroha) as part of the transition from the Hyperledger Foundation to the Linux Foundation’s Decentralized Trust initiative.
+
+The Iroha 2 community is active and growing, with lots of interest from potential users and contributors. Questions are being answered regularly, five issues were assigned to the maintainers among community members, and  the PRs for the ones have been merged since the last update. Project updates are being sent every two weeks to keep everyone connected and informed.
```

Comment:
```suggestion
The Iroha 2 community is active and growing, with lots of interest from potential users and contributors. Questions are being answered regularly, five issues were assigned to the maintainers among community members, and the PRs for these have been merged since the last update. Project updates are being sent every two weeks to keep everyone connected and informed.
```